### PR TITLE
Alex's weather app (required functions are there, but not finished with CSS styling)

### PR DESCRIPTION
**Pull request by Xikitita**
## Netlify link
https://idyllic-maamoul-97c14d.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./style.css">
+    <title>Alex's weather app</title>
+</head>
+<body> 
+<div class="page-top">
+      <div class="typeofweather" id="typeOfWeather">Loading weather...</div>
```
Comment (Tejpex) [inline index.html:11]: Good with a default text!

File: script.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+
+const BASE_URL = "https://api.openweathermap.org/data/2.5/weather?q=Stockholm,Sweden&units=metric&APPID=343b966846b558ccc45becaa3d348154"
```
Comment (Tejpex) [inline script.js:2]: Does the API-key need to be written twice, or could this be shorter?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    fetch(`${BASE_URL}`)
+    .then((response) => {
+        return response.json()
+    })
+    .then((data) => {
+    //variables for sunrise
+    const sunriseData = data.sys.sunrise
+    const sunriseHour = new Date(sunriseData*1000).getHours()
+    const sunriseMinutes = new Date(sunriseData*1000).getMinutes()
+    const sunriseTime = (`${sunriseHour}:${sunriseMinutes}`)
```
Comment (Tejpex) [inline script.js:24]: With this code the time "18:05" is displayed as "18:5" which is somewhat confusing. I had to fix this in my own code using an if statement like this:

`if (sunriseMinute < 10) {
    sunriseMinuteAdjusted = `0${sunriseMinute}`
  } else {
    sunriseMinuteAdjusted = sunriseMinute
  }
const sunriseTime = `${sunriseHour}:${sunriseMinuteAdjusted}``

File: script.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+//Five days weather forecast 
+const getWeatherForecast = () => {
+    fetch(`${FORECAST_URL}`)
+    .then((response) => {
+        return response.json()
+    })
+    .then((data) => {
+    const weekday = [...data.list].filter((data) =>{
+        return data.dt_txt.endsWith("12:00:00")
+    })
```
Comment (Tejpex) [inline script.js:50]: Good way to find only the forecasts from 12 o'clock!

File: script.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+weekday.forEach((data) => {
+    const fromUnix = data.dt * 1000
+    const currentDate = new Date(fromUnix)
+    const fiveDaysList = currentDate.toLocaleDateString('en-US', {weekday: 'short'})
+    const temperatureFiveDays = data.main.temp.toFixed(0)
```
Comment (Tejpex) [inline script.js:56]: Should it be no decimals in the forecast or should it be one decimal here as well?

File: script.js [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+    })
+    .then((data) => {
+    const weekday = [...data.list].filter((data) =>{
+        return data.dt_txt.endsWith("12:00:00")
+    })
+
+weekday.forEach((data) => {
+    const fromUnix = data.dt * 1000
+    const currentDate = new Date(fromUnix)
+    const fiveDaysList = currentDate.toLocaleDateString('en-US', {weekday: 'short'})
```
Comment (Tejpex) [inline script.js:55]: Does this write the weekdays Mon, Tue, Wed etc?
If so, I learned something new :)

**Review by Tejpex — COMMENTED** [review]
Great that the app is working and the code is clean!

Obviously the styling isn't finished, so I have disregarded that part.

File: script.js (hunk 2 above)
Comment (Xikitita) [reply script.js:2]: Very good input. Will look into that!

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Hi Alex, good job submitting your project! Have you tested the app before opening the PR? There is no icons displayed 😢 

Please consider these requirements:
- The app should have: 4-day forecast You are displaying all 5 days when we requested 4 
- the temperature should be rounded to 1 decimal
- the app should follow as close as possible the chosen design

Take a look again at your code carefully and fix the issues pls, thank you ⭐

**Pull request closed without merging**